scripts/src/tests/helpers.ts

```diff
@@ -180,6 +182,26 @@ export async function createOffers() {
 	}
 }
 
+async function createMalformedOffer(num: number) {
+	const offerName = generateId();
+	await createEarn(
+		`${ offerName }_earn${ num }`,
+		"GBOQY4LENMPZGBROR7PE5U3UXMK22OTUBCUISVEQ6XOQ2UDPLELIEC4J",
+		`earn${ num }`, `earn${ num }`, `earn${ num }`, `earn${ num }`, 100, 30, 1, `earn${ num }`, `earn${ num }`, "poll", animalPoll, ["ALL"]
```

Comment:
more line breaks

Reply:
I copied it from the same function above
I'll add line breaks

Reply:
ok